client/src/components/Home/Home.js

```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components/macro";
+
+function Home() {
+  return (
+    <Wrapper>
+      <HomeTitle>AppDiv</HomeTitle>
+      <HomeMenu>Placeholder</HomeMenu>
+      <HomeMenu>Placeholder</HomeMenu>
+    </Wrapper>
+  );
+}
+
+export default Home;
+
+const Wrapper = styled.section`
+  
+  color: white;
+  padding-top: 100px;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  border: 2px solid red;
+`;
+
+const HomeTitle = styled.h1`
```

Comment:
You already have a h1 in your header. More then one is not best practice. Maybe change to h2.